lib/utils/ember.js

```diff
@@ -88,8 +88,15 @@ function isModule(node, element, module) {
     (isLocalModule(node.callee, element) || isEmberModule(node.callee, element, moduleName));
 }
 
-function isDSModel(node) {
-  return isModule(node, 'Model', 'DS');
+function isDSModel(node, filePath) {
+  const isExtended = node.callee.property ? node.callee.property.name === 'extend' : false;
+  let isModuleByPath;
+
+  if (filePath) {
+    isModuleByPath = isModuleByFilePath(filePath, 'model') && isExtended;
```

Comment:
why not put `isExtended` in the condition above instead of applying `&&` to the result? if `isExtended` is false (with the current logic) there wouldn't be any reason to even call `isModuleByFilePath()`